toasty/simple_interface.py

```diff
@@ -0,0 +1,51 @@
+from toasty import builder, collection, pyramid, multi_tan, multi_wcs
+import reproject
+
+
+def toast_fits_list(fits_list, **kwargs):

```

Comment:
Likewise, I think it might be better to call this function `tile_fits`, and have the docstring explain that the first argument can be a single FITS or a list of them. ("Tile" rather than "toast" since we are not necessarily using the TOAST projection ... in fact right now we definitely won't use it, since we can't render TOAST FITS for now!)